src/lib-test/fuzzer.h

```diff
@@ -0,0 +1,16 @@
+#ifndef FUZZER_H
+#define FUZZER_H
+
+#define FUZZ_DATA(args) \
+	int LLVMFuzzerTestOneInput(args); \
```

Comment:
should this be ((args)) ?

Reply:
No, my idea was that it would be used like `FUZZ_DATA(const uint8_t *data, size_t size)` when binary input is wanted to be tested rather than NUL-terminated string.

Reply:
but what's FUZZ_END then?